website/content/en/preview/contributing/development-guide.md

```diff
@@ -21,42 +21,64 @@ The following tools are required for contributing to the Karpenter project.
+#### Container Repository
 
-#### Locally
+For local development on Karpenter you will need a Docker repo which can manage your images for Karpenter components.
+You can use the following command to provision an ECR repository. We recommend using a single "dev" repository for
+development across multiple projects, and to use specific image hashes instead of image tags.
+
+```bash
+aws ecr create-repository \
+    --repository-name dev \
+    --image-scanning-configuration scanOnPush=true \
+    --region "${AWS_DEFAULT_REGION}"
+```
 
-Once you have your environment set up, run the following commands to run the Karpenter Go binary against the Kubernetes cluster specified in your `~/.kube/config`
+Once you have your ECR repository provisioned, configure your Docker daemon to authenticate with your newly created repository.
 
 ```bash
-make run
+export KO_DOCKER_REPO="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/dev"
+aws ecr get-login-password --region "${AWS_DEFAULT_REGION}" | docker login --username AWS --password-stdin "${KO_DOCKER_REPO}"
 ```
 
-#### Inside a Kubernetes Cluster
+#### Kubernetes Cluster
+
+You can refer to the [Getting Started Guide]({{<ref "../getting-started/getting-started-with-karpenter#create-a-cluster" >}}) for details on how to setup a cluster and IAM permissions to deploy Karpenter.
+
+#### Test Dependencies
 
-Once you have your environment set up, to install Karpenter in the Kubernetes cluster specified in your `~/.kube/config`  run the following commands.
+Karpenter requires [Kubebuilder's `envtest` dependencies](https://book.kubebuilder.io/reference/envtest.html) to run functional testing. Run the following command to install the test dependencies.
 
 ```bash
-make apply # Install Karpenter
-make delete # Uninstall Karpenter
+make toolchain # install test dependencies
 ```
 
-### Developer Loop
+#### Environment Variables
 
-* Make sure dependencies are installed
-    * Run `make codegen` to make sure yaml manifests are generated
-    * Run `make toolchain` to install cli tools for building and testing the project
-* You will need a personal development image repository (e.g. ECR)
-    * Make sure you have valid credentials to your development repository.
-    * `$KO_DOCKER_REPO` must point to your development repository
-    * Your cluster must have permissions to read from the repository
+Karpenter requires certain environment variables be set to use the `Makefile`. Ensure you've set the following values.
+
+```bash
+export AWS_ACCOUNT_ID=<account-id>
+export AWS_DEFAULT_REGION=<region>
+export AWS_SDK_LOAD_CONFIG=true # allows global config to be passed through on local runs
+```
 
 ### Build and Deploy
```

Comment:
Are there prerequisites you need to mention here before you get to the prerequisites section below? Will the reader just know to get tools like `make`?

Reply:
Yeah, I think that having `make` on your system should be seen as an implict requirement. Not sure that we need to call that one out explicitly.